### PR TITLE
refac: Read from calculations unity catalog

### PR DESCRIPTION
**Pull request by johevemi**
# Description

## Pull-request quality

- [ ] The title adheres to [this guide](https://github.com/Mech0z/GitHubGuidelines)
- [ ] Tests are written and executed locally
- [ ] Subsystem tests have been tested (by manually deploying to `dev_002`)

### CODE REVIEW

File: source/databricks/calculation_engine/package/databases/wholesale_internal/repository.py [hunk 1]
```diff
@@ -29,13 +30,17 @@ def __init__(
...
+        calculations_table_name: str | None = None,
     ) -> None:
         self._spark = spark
         self._catalog_name = catalog_name
         self._grid_loss_metering_points_table_name = (
             grid_loss_metering_points_table_name
             or WholesaleInternalDatabase.GRID_LOSS_METERING_POINTS_TABLE_NAME
         )
+        self._calculations_table_name = (
+            calculations_table_name or HiveBasisDataDatabase.CALCULATIONS_TABLE_NAME
```
Comment (JesperGomes) [inline source/databricks/calculation_engine/package/databases/wholesale_internal/repository.py:42]: hive?

Comment (johevemi) [reply source/databricks/calculation_engine/package/databases/wholesale_internal/repository.py:42]: missed that

File: source/databricks/calculation_engine/package/databases/wholesale_internal/repository.py [hunk 2]
```diff
@@ -29,13 +30,17 @@ def __init__(
         spark: SparkSession,
         catalog_name: str,
         grid_loss_metering_points_table_name: str | None = None,
+        calculations_table_name: str | None = None,
```
Comment (BjarkeMeier) [inline source/databricks/calculation_engine/package/databases/wholesale_internal/repository.py:33]: When do you need to be able to overwrite this?

Comment (johevemi) [reply source/databricks/calculation_engine/package/databases/wholesale_internal/repository.py:33]: idk

**Review by BjarkeMeier — DISMISSED** [review]

File: source/databricks/calculation_engine/package/databases/wholesale_internal/repository.py (hunk 2 above)
Comment (BjarkeMeier) [reply source/databricks/calculation_engine/package/databases/wholesale_internal/repository.py:33]: Then I suggest that you remove this parameter.

Comment (johevemi) [reply source/databricks/calculation_engine/package/databases/wholesale_internal/repository.py]: will do

**Review by JesperGomes — APPROVED** [review]

**Pull request merged**